Makefile

```diff
@@ -174,11 +174,12 @@ run-frontend: deps-js ## Fetch js dependencies and watch frontend for rebuild
 .PHONY: test-go
 test-go: test-go-unit test-go-integration
 
+### TODO: temporarily run only the failing test (fails in PR only)
 .PHONY: test-go-unit
 test-go-unit: ## Run unit tests for backend with flags.
 	@echo "test backend unit tests"
-	go list -f '{{.Dir}}/...' -m | xargs \
-	$(GO) test -short -covermode=atomic -timeout=30m 
+	go list -f '{{.Dir}}/...' ./pkg/apiserver/storage/file | xargs \
```

Comment:
Oops! Made an unintentional change here. Will PR something to roll it back soon.